### PR TITLE
Added "summary vs snapshot" to FAQ

### PR DESCRIPTION
**Pull request by agarwal-navin**
Added a section called "Summarization" to the FAQ where we can add details about the summary proces. To begin with, added "Summary vs Snapshot" sub-section which have been asked by customers.

### CODE REVIEW

File: docs/content/docs/faq.md [hunk 1]
```diff
@@ -242,3 +242,21 @@ This also applies to Blazor, Xamarin, MAUI, and other mobile frameworks.
...
+
+## Summarization
+
+### Summaries vs Snapshots
+
+#### Why do we need summaries and snapshots?
+Summaries capture the state of a container at a point in time. Snapshots allow new clients to quickly catch up to recent state. Without them, a client would have to apply every operation in the op log, even if those operations no longer affected the current state (e.g. op 1 inserts 'h' and op 2 deletes 'h'). For very large op logs, this would be very expensive both for the clients to process and to download from the service.
+Instead, when a client joins a collaborative document, they can download a snapshot of the container state, and simply process new operations from that point forward.
+
+#### Summary vs Snapshot
```
Comment (agarwal-navin) [inline docs/content/docs/faq.md:254]: Maybe I should name this section "Differences". Any thoughts?

File: docs/content/docs/faq.md [hunk 2]
```diff
@@ -242,3 +242,24 @@ This also applies to Blazor, Xamarin, MAUI, and other mobile frameworks.
 ### What browsers are supported?
 
 {{% include file="_includes/browsers.md" %}}
+
+## Summarization
+
+### Summaries vs Snapshots
```
Comment (kashms) [inline docs/content/docs/faq.md:248]: I suggest we remove this header, don't think we need a separate sub-section heading. Instead use the triple # for questions below. Looking over the other questions in the FAQ, the rest of them are also triple #

File: docs/content/docs/faq.md [hunk 3]
```diff
@@ -242,3 +242,24 @@ This also applies to Blazor, Xamarin, MAUI, and other mobile frameworks.
...
+
+#### Why do we need summaries and snapshots?
```
Comment (kashms) [inline docs/content/docs/faq.md:250]: Change to ###

File: docs/content/docs/faq.md (hunk 1 above)
Comment (kashms) [reply docs/content/docs/faq.md:255]: Also change to ###
Yea, we should change the Q to "What's the difference between summaries and snapshots?"

**Review by kashms — CHANGES_REQUESTED** [review]
Some small format/text changes

File: docs/content/docs/faq.md (hunk 1 above)
Comment (agarwal-navin) [reply docs/content/docs/faq.md]: Updated.

File: docs/content/docs/faq.md (hunk 2 above)
Comment (agarwal-navin) [reply docs/content/docs/faq.md]: Good call. Updated.

**Review by kashms — APPROVED** [review]
LGTM 🚢

**Pull request merged**